bump.lua

```diff
@@ -696,19 +742,23 @@ function World:move(item, goalX, goalY, filter)
 end
 
```

Comment:
I added a new method `World:projectMove()`. This is essentially `World:check()`, except you can use it to check collisions for items which aren't actually stored inside the world.

I've found this **incredibly** practical for optimization, because the fewer things which are stored in the world, the faster collision detection is.

An example of how you could use this: Bullets need to collide with Actors; however, Actors don't necessarily need to collide with Bullets. So, we insert actors into the bump world, but we keep track of the bullets some other way. We can still have the bullets interact with the actors by using `World:projectMove(bullet, ...)`.